sklearn/calibration.py

```diff
@@ -25,6 +26,197 @@
 from .svm import LinearSVC
 from .model_selection import check_cv
 from .metrics.classification import _check_binary_probabilistic_predictions
+from .metrics.pairwise import euclidean_distances
+from .metrics.ranking import roc_curve
+
+
+class OptimalCutoffClassifier(BaseEstimator, ClassifierMixin):
+    """Optimal cutoff point selection.
```

Comment:
I'm not sure having ``Optimal`` in the name is a good idea. Maybe just ``CutoffClassifier``? There is no description in the docstring on what it does.